product/timelines/miq_reports/tl_events_daily.yaml

```diff
@@ -36,6 +36,7 @@ cols:
 - container_namespace
 - container_name
 - container_replicator_name
+- middleware_name
```

Comment:
Actually, it is generic.  The idea here is to be able to link to various object types and not have to add rows for each different sort of inventory object that gets added to the MW provider.  The event type can be associated with an object type (for example, 'hawkular_deployment.error' is relevant to MiddlewareServer).  That is why the fields seem generic, middleware_id, middleware_name, middleware_type.  And in the bubble you'll see 'Middleware Resource: <name as link>'